src/module/behaviors.js

```diff
@@ -0,0 +1,230 @@
+    likeBtn.addEventListener('click', async () => {
+      const gameId = game.id;
+      await likeGame(gameId);
+      const updatedLikes = await getLikes();
+      span.textContent = `${updatedLikes[gameId]} likes`;
+    });
+  });
+  const commentsBtn = document.querySelectorAll('.comments-btn');
+  const disabled = false; // flag to keep track of whether the action is disabled or not
+
+  commentsBtn.forEach((button, index) => {
+    button.addEventListener('click', async () => {
+      if (disabled) {
+        return; // exit the function if the action is disabled
+      }
+
+      const commentsCont = document.querySelector('.comments-' + `${games[index].id}`);
+
+      // Fetch comments for item
+      const commentsResponse = await fetch(`${invAPI}/comments?item_id=${games[index].id}`);
+      const commentsData = await commentsResponse.json();
+
+      // Format comments data into HTML
+      const commentsHTML = commentsData.map((comment) => `<ul class="commentsUl">
+    <li class="comment-date"> ${comment.creation_date} </li>
+    <li class="comment-user"> ${comment.username}: </li>
+    <li class="comment-comment"> ${comment.comment} </li>
+              </ul>`).join('');
+
+      const newComment = document.createElement('div');
+      newComment.innerHTML = commentsHTML;
+
+      // Clear existing comments
+      commentsCont.innerHTML = '';
+
+      commentsCont.appendChild(newComment);
+    });
+  });
+
+  let canSubmitComment = true;
+  const submitBtn = document.querySelectorAll('.submit-btn');
+
+  submitBtn.forEach((button, index) => {
+    button.addEventListener('click', async (event) => {
+      event.preventDefault();
+      if (!canSubmitComment) return;
+
+      const form = button.closest('form');
+      const name = form.querySelector('.userInput').value;
+      const comment = form.querySelector('.commentInput').value;
+      const gameId = games[index].id;
+
+      if (name.trim() === '' || comment.trim() === '') {
+        return;
+      }
+
+      canSubmitComment = false;
+      setTimeout(() => {
+        canSubmitComment = true;
+      }, 5000); // wait 5 seconds before enabling comments again
+
+      await addComment(gameId, name, comment);
+
+      const commentsCont = document.querySelector('.comments-' + `${gameId}`);
+      const commentsData = await getComments(gameId);
+      const commentsHTML = commentsData.map((comment) => `<ul class="commentsUl">
+      <li class="comment-date"> ${comment.creation_date} </li>
+      <li class="comment-user"> ${comment.username}: </li>
+      <li class="comment-comment"> ${comment.comment} </li>
+    </ul>`).join('');
+      commentsCont.innerHTML = commentsHTML;
+
+      // clear the input fields
+      form.querySelector('.userInput').value = '';
+      form.querySelector('.commentInput').value = '';
+    });
+  });
+};
+
+export const init = () => {
+  fetchData().then((data) => {
+    renderGameCards(data);
+  });
+};
+
+init();
```

Comment:
- [ ] Good job working on other functionalities, but please it is important that you implement a counter functionality and include it on every page. The counter functionality should count the number of items in the list on that page.
- [ ] There should be a counter functionality on the homepage that displays the number of items on that page.
- [ ] Also, there should be another one in the comments popup that shows the number of comments for that item.
- [ ] Please also write a unit test to test out the counter functionality.